src/backend/columnar/cstore_tableam.c

```diff
@@ -1292,6 +1305,49 @@ ColumnarTableAMObjectAccessHook(ObjectAccessType access, Oid classId, Oid object
 }
 
 
+/*
+ * Utility hook for columnar tables.
+ */
+static void
+ColumnarProcessUtility(PlannedStmt *pstmt,
+					   const char *queryString,
+					   ProcessUtilityContext context,
+					   ParamListInfo params,
+					   struct QueryEnvironment *queryEnv,
+					   DestReceiver *dest,
+					   QueryCompletionCompat *completionTag)
```

Comment:
Don't different versions have different signatures for the hook?

Reply:
This is abstracted by definition of `QueryCompletionCompat` in `version_compat.c`.